docs/lit/examples/1_Examples.jl

```diff
@@ -52,6 +52,67 @@ plot(y, ylabel=P"mass in kilograms")
 
 plot(y, ylabel="")
 
+# ### Unit formatting
+
+# If you prefer some other formatting over the round parentheses, you can
+# supply a keyword `unitformat`, which can be a number of different things: A
+# `Symbol` representing a standard case (see example below), a `Char`, a
+# `String`, or a `Tuple` (of `Char`s or `String`s), which will be inserted
+# arround the label and unit depending on the length of the tuple.
+
+plot(y, ylabel="mass", unitformat=:square)
+
+# For *extreme* customizability, you can also supply a function that turns two
+# arguments (label, unit) into a string.
+
+plot(y, ylabel="mass", unitformat=(l, u)->string("\$\\frac{", l, "}{\\mathrm{", u, "}}\$"))
+
+# A complete list of options:
+labelformatter(l, u) = string(u, ' ', l)
+unitformats = [
+             [
+              :round,
+              :square,
+              :curly,
+              :angle,
+              :slash,
+              :slashround,
+              :slashsquare,
+              :slashcurly,
+              :slashangle,
+              :verbose,
+             ],
+             [
+              ", in ",
+              (", in (", ")"),
+              ("[", "] = (", ")"),
+              ':',
+              ('$', '$'),
+              (':', ':', ':'),
+             ],
+             [
+              nothing,
+              labelformatter,
+              false,
+              true,
+             ],
+            ]
+plot([
+      plot([
+            plot(y, 1:10, xlabel="mass", title=repr(s),
+                 unitformat=s) for s in unitformats[1]
+           ]..., layout=(2, 5)),
+      plot([
+            plot(y, 1:10, xlabel="mass", title=repr(s),
+                 unitformat=s) for s in unitformats[2]
+           ]..., layout=(2, 3)),
+      plot([
+            plot(y, 1:10, xlabel="mass", title=repr(s),
+                 unitformat=s) for s in unitformats[3]
+           ]..., layout=(1, 4)),
+     ]..., layout=(3, 1), size=(1200, 800),
+    )
+
```

Comment:
I think we can turn this into something cleaner with much shorter code cells, e.g., something like this (not tested, likely contains typos/errors):

```julia
# ### Unit formatting

# If you prefer some other formatting over the round parentheses, you can
# supply a keyword `unitformat`, which can be a number of different things.

# `unitformat` can be a boolean or `nothing`:

plot([plot(y, ylab="mass", title=repr(s), unitformat=s) for s in (nothing, true, false)]...)

# `unitformat` can be a symbol from one of the predefined symbols that you can check with

URsymbols = keys(UnitfulRecipes.UNIT_FORMATS)

# and allow you one of these unit formats:

plot([plot(y, ylab="mass", title=repr(s), unitformat=s) for s in URsymbols]...)

# `unitformat` can also be a `Char`, a `String`, or a `Tuple` 
# (of `Char`s or `String`s), which will be inserted
# arround the label and unit depending on the length of the tuple

URtuples = [", in ", (", in (", ")"), ("[", "] = (", ")"), ':', ('$', '$'), (':', ':', ':')],
plot([plot(y, ylab="mass", title=repr(s), unitformat=s) for s in URtuples]...)

# And for *extreme* customizability, `unitformat` can a function that takes two
# arguments, `(label, unit)`, and returns a string:

plot(y, ylab="mass", unitformat=(l, u)->string("\$\\frac{", l, "}{\\mathrm{", u, "}}\$"))
```

Reply:
That looks a lot better. I just changed a couple of formulations that weren't clear to me.